tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/MsTestIntegration.cs

```diff
@@ -186,4 +195,62 @@ internal static bool ShouldSkip<TTestMethod>(TTestMethod testMethodInfo, out boo
         isForcedRun = itrShouldSkip && isUnskippable;
         return itrShouldSkip && !isUnskippable;
     }
+
+    internal static TestModule GetOrCreateTestModuleFromTestAssemblyInfo<TAsmInfo>(TAsmInfo testAssemblyInfo, string assemblyName = null)
+        where TAsmInfo : ITestAssemblyInfo
+    {
+        if (testAssemblyInfo.Instance is not { } objTestAssemblyInfo)
+        {
+            return default;
```

Comment:
Urgh, this could do with some `#nullable enable` 😄

Reply:
well I'm doing this because IDuckType.Instance is nullable so before accessing the proxy I want to make sure is not going to crash because a missing Instance

Reply:
oh, I totally get it, but the file doesn't have nullable reference types, so it's not clear the method returns `null` 😛